Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Image/StreamingImage.h

```diff
@@ -118,6 +109,25 @@ namespace AZ
             //! Returns the average color of this image (alpha-weighted in case of 4-component images).
             Color GetAverageColor() const;
 
+            //! Returns the image's streaming priority 
+            using Priority = uint64_t;
+            Priority GetStreamingPriority() const;
+
+            //! Set the image's streaming priority 
+            void SetStreamingPriority(Priority priority);
+            
+            //! Returns whether the image has mipchains which can be evicted from device memory
```

Comment:
Nit: "mips" instead of "mipchains"

Reply:
It's actually "mipchains". The current RPI::StreamingImage's api only provides trim and expand functions for mipchain but not mip.